### DISCUSSION TITLE
*.blade.php

### DISCUSSION THREAD

**Discussion by ghost** [tighten/jigsaw#612] [General]
This is something I kind of need to make sure. I know most of the time when you work with jigsaw and most likely the blade templates are mainly use 
```
source
   about.blade.php
   index.blade.php
```
I just found while working on this, the system allows us to rename `index.blade.php` to `index.md`.  I also this is okay too right. I just wanted someone to confirm.
```
source
   404.md
   about.md
   index.md
```

**Comment by bakerkretzmar** [comment]
Yep, that should work fine, and if you need Blade in your Markdown `about.blade.md` will even work too.
